src/Application/PresenterFormatter/PresenterFormatterManager.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ */
+
+namespace Nette\Application\PresenterFormatter;
+
+use Nette\Application\PresenterFormatterException;
+
+/**
+ * Responsible for manage multiple IPresenterFormatter
+ */
+class PresenterFormatterManager implements IPresenterFormatter
```

Comment:
This is typically named Chain, so e.g. PresenterFormatterChain, with addFormatter method.